data/brands/shop/supermarket.json

```diff
@@ -9476,6 +9477,19 @@
         "shop": "supermarket"
       }
     },
+    {
+      "displayName": "טיב טעם בעיר",
+      "locationSet": {"include": ["il"]},
+      "tags": {
+        "alt_name": "Tiv Ta'am in The City",
+        "brand": "Tiv Ta'am in The City",
+        "brand:wikidata": "Q122189282",
```

Comment:
https://www.wikidata.org/wiki/Q122189282 was speedily deleted as not meeting [Wikidata notability](https://www.wikidata.org/wiki/Wikidata:Notability). Please see https://github.com/osmlab/name-suggestion-index/pull/8475#issuecomment-1665543134 and https://www.wikidata.org/wiki/Wikidata:Guide_to_requests_for_undeletion for more information.

Reply:
we already have this place with brand Tiv Ta'am / brand:wikidata: Q2903718

Reply:
It's not ideal, but we can't control what Wikidata does.